README.md

```diff
@@ -66,14 +66,31 @@ a wrapper.
 
 #### Example
 
-```js
+```javascript
```

Comment:
was the language set by remark to use the expanded name? i dont really care which is used, but was curious about the change

Reply:
yup this is what remark printed out